### PR TITLE
initial ebuild for sci-biology/cnrun

### PR DESCRIPTION
**Pull request by hmmr**
CNrun is a neuronal network simulator, with these features:

 * a conductance- and rate-based Hodgkin-Huxley neurons, a Rall and
   Alpha-Beta synapses;
 * a 6-5 Runge-Kutta integration method: slow but precise, adjustable;
 * Poisson, Van der Pol, Colpitts oscillators and interface for
   external stimulation sources;
 * NeuroML network topology import/export;
 * logging state variables, spikes;
 * implemented as a Lua module, for scripting model behaviour (e.g.,
   to enable plastic processes regulated by model state);
 * interaction (topology push/pull, async connections) with other
   cnrun models running elsewhere on a network, with interactions
   (planned).

### CODE REVIEW

**Comment by hmmr** [comment]
That's the second and last one.

**Comment by jlec** [comment]
Please use repoman or repo-commit to commit. Plain git commands are not allowed.

**Comment by marbre** [comment]
Using repoman would also result in updating a Manifest update which will solve the QA issues travis detected:

> digest.missing 1
> /home/travis/build/gentoo-science/sci/travis-overlay/sci-biology/cnrun::cnrun-2.0.2.tar.xz

**Comment by hmmr** [comment]
Well.. I have previously packaged this stuff for Debian, and honestly, I no longer think making their lintian pleased is the hardest task of all maintainer chores.

For the latest commits, I checked my ebuilds with
```$ repoman -d -x --digest=y --echangelog=force full```
to fix any remaining issues.

File: sci-biology/aghermann/ChangeLog [hunk 1]
```diff
@@ -2,5 +2,8 @@
 # Copyright 1999-2015 Gentoo Foundation; Distributed under the GPL v2
 # $Header: $
 
+  18 Feb 2015; HMMR <hmmr@gentoo.org> aghermann-1.0.3.ebuild:
```
Comment (junghans) [inline sci-biology/aghermann/ChangeLog]: Please fix name and email using
```
export ECHANGELOG_USER="First Last <123@456>"
```

File: sci-biology/cnrun/ChangeLog [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# ChangeLog for sci-biology/cnrun
+# Copyright 1999-2015 Gentoo Foundation; Distributed under the GPL v2
+# $Header: $
+
+  18 Feb 2015; HMMR <hmmr@gentoo.org> cnrun-2.0.2.ebuild:
```
Comment (junghans) [inline sci-biology/cnrun/ChangeLog]: Same here!

**Comment by jlec** [comment]
@hmmr just use "repoman commit" in future to commit. Currently we have a merge conflict. Could you please rebase onto the master branch?

**Comment by hmmr** [comment]
Here's some squash+rebase for cnrun. For aghermann (part of which has already been merged), it seems a separate PR is needed; will push after cnrun gets merged.

**Comment by hmmr** [comment]
Tada!

**Comment by jlec** [comment]
Thanks for your contribution!!

**Pull request merged**